Week-10/src/App.js

```diff
@@ -0,0 +1,46 @@
+import Main from "./main";
+import "./styles.css";
+import { useState } from "react";
+
+export default function App() {
+  const [show, setShow] = useState(false);
+  const [type, setType] = useState('success');
+  const [toastMsg, setToastMsg] = useState("")
+  const openTost = (msg = '', type = 'success', time = 3000) => {
+    setToastMsg(msg)
+    setShow(true)
+    setType(type)
+    setTimeout(() => {
+      closeTost()
+    }, time)
+  }
+  const closeTost = () => {
+    setToastMsg("")
+    setShow(false)
+    setType('success')
+  }
+  return (
+    <>
+      <Main openTost={openTost} closeTost={closeTost} />
+      <div className="toast-container position-fixed bottom-0 end-0 p-3">
+        <div
+          id="liveToast"
+          className={"toast" + (show ? ' show' : '') + (type === 'success' ? ' text-bg-success' : type === 'error' ? ' text-bg-danger' : '')}
```

Comment:
we can create a  class variable  at the top .

toastClass = "toast" + (show ? ' show' : '') + (type === 'success' ? ' text-bg-success' : type === 'error' ? ' text-bg-danger' : '';
ans use this class.